# Introduction

Open Targets Genetics \(OT Genetics\) is a portal highlighting variant-centric statistical evidence to allow prioritisation of candidate causal variants at trait-associated loci, and identification of potential drug targets. 

It is well established that proximity is a poor basis on which to prioritise 'causal' genes at a trait-associated locus.  Rather, integrating functional and biological data from multiple heterogeneous sources allows functionally implicated genes to be highlighted. Our portal aggregates and merges genetic associations curated from literature and newly-derived loci from UK Biobank with \(open source\) functional genomics, chromatin conformation, epigenetic, and expression data \(e.g., eQTLs, regulatory elements\), and applies statistical fine-mapping across thousands of trait-associated loci, to resolve association signals and link each variant to its proximal and distal target gene\(s\), using a single evidence score. Integrated cross-trait colocalisation analyses and cross-linking to details of pharmaceutical compounds extend the capacity of the platform to explore drug repositioning opportunities and shared genetic architecture. Take a look at our [Data Pipeline](our-approach/data-pipeline.md) and [Data Sources](our-approach/data-sources.md#overview-of-data-sources) for more detailed information.  

Whatever your starting point - [gene](how-to-use-ot-genetics-starting-with.../gene-target.md), [trait](how-to-use-ot-genetics-starting-with.../trait-or-disease.md) or [variant ](how-to-use-ot-genetics-starting-with.../variant.md)- OT Genetics enables detailed biological insight  and causal gene prioritisation, and informs target decision making.  It can be used to answer specific biological and target hypotheses, or as an exploratory tool - for example, to rapidly prioritise genes at associated loci in a new GWAS.  Get started now by navigating to one of our tutorials.     

{% hint style="success" %}
OT is wholly open-source, freely available to academic and for-profit users alike.  Please **clearly acknowledge use of this resource** in published works, by linking genetics.opentargets.io and citing Open Targets' flagship paper:  '[Open Targets:  a platform for therapeutic target identification and validation](https://doi.org/10.1093/nar/gkw1055)'   
{% endhint %}

### About Open Targets

Open Targets is a academic-industrial partnership which aims to address the inefficiencies of standard drug development pipelines, and expedite the translation of research findings to licensed drugs.  We integrate large-scale genetics and genomics with drug information and many other public and proprietary data sources, influencing the way drug targets are identified and prioritised. 

Using human cell models and genome editing, we are systematically generating new data to identify drug targets for three main therapeutic areas: oncology, immunology, and neurodegeneration. The Open Targets Platform \([targetvalidation.org](www.targetvalidation.org)\) enables users to investigate links between genes, pathways, diseases and existing drugs.  We created the OT Genetics Portal as a complementary, variant-focussed resource aimed at geneticists, to flexibly integrate all trait and functional annotations for human polymorphic variants.  

### Our Partners

[Open Targets](www.opentargets.orgâ€‹) is a joint collaboration of the Wellcome Sanger Institute, European Molecular Biology Laboratory-European Bioinformatics Institute \(EMBL-EBI\), GlaxoSmithKline plc, Takeda Ltd, Celgene Corporation and Biogen Inc.

### Copyright

Copyright 2014-2018 Biogen, Celgene Corporation, EMBL - European Bioinformatics Institute, GlaxoSmithKline, Takeda Pharmaceutical Company and Wellcome Sanger Institute.  This software was developed as part of the Open Targets project \(www.opentargets.org\) 

Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except in compliance with the License. You may obtain a copy of the License at

[http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0)

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.



