# Introduction

### Overview

The [**Open Targets Genetics Portal**](https://genetics.opentargets.org/) is a tool highlighting variant-centric statistical evidence to allow both prioritisation of candidate causal variants at trait-associated loci and identification of potential drug targets.

It is well established that proximity is a poor basis on which to prioritise 'causal' genes at a trait-associated locus. Rather, integrating functional and biological data from multiple heterogeneous sources allows functionally implicated genes to be highlighted. Our portal aggregates and merges genetic associations curated from both literature and newly-derived loci from UK Biobank, which contains functional genomics data \(e.g. chromatin conformation, chromatin interactions\) and quantitative trait loci \(eQTLs and pQTLs\). We apply statistical fine-mapping across thousands of trait-associated loci to resolve association signals, and link each variant to its proximal and distal target gene\(s\), using a single evidence score. Integrated cross-trait colocalisation analyses and linking to detailed pharmaceutical compounds extend the capacity of the Open Targets Genetics Portal to explore drug repositioning opportunities and shared genetic architecture. Take a look at our [data pipeline](our-approach/data-pipeline.md) and [data sources](our-approach/data-sources.md#overview-of-data-sources) for more detailed information.

Whatever your starting point - [gene](how-to-use-open-targets-genetics-starting-with/gene-target.md), [trait](how-to-use-open-targets-genetics-starting-with/trait-or-disease.md) or [variant ](how-to-use-open-targets-genetics-starting-with/variant.md)- the Genetics Portal enables detailed biological insight and causal gene prioritisation and informs target decision making. It can be used to answer specific biological and target hypotheses or as an exploratory too \(e.g. to prioritise genes at associated loci in a new GWAS\). 

{% hint style="success" %}
The Open Targets Genetics Portal is an open source, freely available research tool that is available for both academic and commercial purposes. If you use our data or our computation pipelines in your work, please cite [our latest publication](citation.md#latest-publication). 
{% endhint %}

