# Introduction

### About the Open Targets Genetics Portal

The [**Open Targets Genetics Portal**](https://genetics.opentargets.org/) is a tool highlighting variant-centric statistical evidence to allow both prioritisation of candidate causal variants at trait-associated loci and identification of potential drug targets.

It is well established that proximity is a poor basis on which to prioritise 'causal' genes at a trait-associated locus. Rather, integrating functional and biological data from multiple heterogeneous sources allows functionally implicated genes to be highlighted. Our portal aggregates and merges genetic associations curated from both literature and newly-derived loci from UK Biobank, which contains functional genomics data \(e.g. chromatin conformation, chromatin interactions\) and quantitative trait loci \(eQTLs and pQTLs\). We apply statistical fine-mapping across thousands of trait-associated loci to resolve association signals, and link each variant to its proximal and distal target gene\(s\), using a single evidence score. Integrated cross-trait colocalisation analyses and linking to detailed pharmaceutical compounds extend the capacity of the Open Targets Genetics Portal to explore drug repositioning opportunities and shared genetic architecture. Take a look at our [data pipeline](our-approach/data-pipeline.md) and [data sources](our-approach/data-sources.md#overview-of-data-sources) for more detailed information.

Whatever your starting point - [gene](how-to-use-open-targets-genetics-starting-with/gene-target.md), [trait](how-to-use-open-targets-genetics-starting-with/trait-or-disease.md) or [variant ](how-to-use-open-targets-genetics-starting-with/variant.md)- the Genetics Portal enables detailed biological insight and causal gene prioritisation and informs target decision making. It can be used to answer specific biological and target hypotheses or as an exploratory too \(e.g. to prioritise genes at associated loci in a new GWAS\). 

{% hint style="success" %}
The Open Targets Genetics Portal is an open-source tool that is freely available to academic and industry users. 

Please **clearly acknowledge use of this resource** in your published works, by citing our latest paper published in October 2020 by Ghoussaini et al, [Open Targets Genetics: systematic identification of trait-associated genes using large-scale genetics and functional genomics](https://doi.org/10.1093/nar/gkaa840)
{% endhint %}

[Open Targets](https://www.opentargets.org/) is an academic-industrial partnership which aims to address the inefficiencies of standard drug development pipelines, and expedite the translation of research findings to licensed drugs. We integrate large-scale genetics and genomics with drug information to influence the way drug targets are identified and prioritised.

Using human cell models and genome editing, we have been systematically generating new data to identify drug targets for three main therapeutic areas: oncology, immunology, and neurodegeneration. Our [Open Targets Platform](https://www.targetvalidation.org) enables users to investigate associations between genes and diseases based on germline variants, somatic mutations, affected pathways, drugs, differential expression data, text mining, and animal models. We created Open Targets Genetics as a complementary, variant-focussed resource aimed at geneticists, to flexibly integrate all trait and functional annotations for human polymorphic variants.

### Our Partners

Open Targets is a joint collaboration between Bristol Myers Squibb, the European Molecular Biology Laboratory - European Bioinformatics Institute \(EMBL-EBI\), GSK, Sanofi, Takeda, and the Wellcome Sanger Institute.

### Copyright

Copyright 2014-2020 Bristol Myers Squibb, EMBL - European Bioinformatics Institute, GlaxoSmithKline, Takeda Pharmaceutical Company, Sanofi S.A., and Wellcome Sanger Institute. This software was developed as part of the Open Targets project \(www.opentargets.org\)

Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except in compliance with the License. You may obtain a copy of the License at

[http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0)

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.

