# Genetic Variation

#### **Variant index**

* [GnomAD v2.1](https://gnomad.broadinstitute.org/)

#### **Disease-associated loci sources (V2D)**

* [GWAS Catalog curated studies](https://www.ebi.ac.uk/gwas/)
* [GWAS Catalog summary statistics](https://www.ebi.ac.uk/gwas/summary-statistics)
* [Neale lab UK Biobank summary statistics](http://www.nealelab.is/uk-biobank/)
* [SAIGE UK Biobank summary statistics](https://www.ncbi.nlm.nih.gov/pubmed/30104761)
* [FinnGen summary statistics](https://r5.finngen.fi/)

#### **Haplotype reference samples**

* [1000 Genomes Phase 3](http://www.internationalgenome.org/category/phase-3/) - used for LD expansion
* [UK Biobank genotypes](https://www.ukbiobank.ac.uk/) - used for conditional analysis in top-loci detection, fine-mapping and colocalisation analyses ([approved application](https://www.ukbiobank.ac.uk/2019/04/using-genetic-data-in-drug-target-discovery-and-validation/)).

#### **Variant functional predictions**

* _In silico_ functional prediction
  * [Variant-Effect Predictor (VEP)](https://www.ncbi.nlm.nih.gov/pubmed/27268795): Most severe transcript consequence(s)
